.github/sample-test-config.yaml

```diff
@@ -0,0 +1,28 @@
+# Test configuration file for running performance benchmarks
+# See: https://docs.meshery.io/guides/performance-management#running-performance-benchmarks-through-mesheryctl
+test:
+  smp_version: v0.0.1
+  
+  # The name of the test
+  name: Sample Performance Test
+  labels: {}
+  
+  # Test configuration to be defined here
+  clients:
+    - internal: false
+      load_generator: fortio
+      protocol: 1
+      connections: 2
+      rps: 10
+      headers: {}
+      cookies: {}
+      body: ''
+      content_type: ''
+      endpoint_urls:
+        - 'https://www.cncf.io/'
```

Comment:
Let's change this to meshery.io

Reply:
Didn't want to cause any issues with this traffic 😅

Reply:
Will make the change if needed in a subsequent PR with more tests.

Reply:
```suggestion
        - 'https://meshery.io'
```